src/xcode/ENA/ENA/Source/Workers/Store/SecureStore.swift

```diff
@@ -361,7 +361,9 @@ extension SecureStore: PrivacyPreservingProviding {
 	var isPrivacyPreservingAnalyticsConsentGiven: Bool {
 		get { kvStore["isPrivacyPreservingAnalyticsConsentGiven"] as Bool? ?? false }
 		set { kvStore["isPrivacyPreservingAnalyticsConsentGiven"] = newValue
-			userData = nil
+			if newValue == false {
```

Comment:
Swiftlint is not complaining that `kvStore["isPrivacyPreservingAnalyticsConsentGiven"] = newValue` is not positioned in the line below? Interesting.

Reply:
no, swiftline is fine here :)